tests/tap_tester/test_mixpanel_all_fields_pagination.py

```diff
@@ -152,3 +152,4 @@ def test_run(self):
         # Pagination test for EU residency server
         self.eu_residency = True
         self.pagination_test_run()
+
```

Comment:
Yeah, created env variable for export_events param and also adjusted start date since data was missing for export stream. So, I had to generate test data for export stream and accordingly adjusted the start date